.github/workflows/trigger-circle-ci-workflow.yml

```
@@ -33,24 +68,24 @@ jobs:
       - if: github.event_name == 'pull_request_target' && (contains(github.event.pull_request.labels.*.name, 'ci:docs'))
-        run: echo "workflow=docs" >> $GITHUB_ENV
+        run: echo "workflow=docs" >> $GITHUB_OUTPUT
       - if: github.event_name == 'push' || contains(github.event.pull_request.labels.*.name, 'ci:merged')
-        run: echo "workflow=merged" >> $GITHUB_ENV
+        run: echo "workflow=merged" >> $GITHUB_OUTPUT
       - if: github.event_name == 'pull_request_target' && (contains(github.event.pull_request.labels.*.name, 'ci:daily'))
-        run: echo "workflow=daily" >> $GITHUB_ENV
+        run: echo "workflow=daily" >> $GITHUB_OUTPUT
     outputs:
-      workflow: ${{ env.workflow }}
+      workflow: ${{ steps.get-parameters.outputs.workflow }}
```

actionlint (1.7.12)

[error] property "get-parameters" is not defined in object type {}

(expression)